src/test/java/org/ow2/proactive/catalogclient/service/CatalogObjectServiceTest.java

```diff
@@ -123,32 +120,27 @@ public void testThatGetRequestReturnTheCatalogObject() {
 
     @Test
     public void testGetRequestWithAResourceWithOneReplacement() throws IOException {
-        when(serviceConfiguration.getCatalogURL()).thenReturn(CATALOG_URL);
         when(remoteObjectService.sendRequest(CATALOG_RESOURCE_URL_RAW,
                                              String.class)).thenReturn(getWorkflowString("workflows/workflowWithPAGetFromURL.xml"));
         when(remoteObjectService.sendRequest(REMOTE_URL, String.class)).thenReturn(REMOTE_VALUE);
 
-        String replacedWorkflow = catalogObjectService.getResource(3, "bobot", true);
+        String replacedWorkflow = catalogObjectService.getResolvedCatalogObject(CATALOG_URL, 3, "bobot", true);
```

Comment:
bobot, we can declare it in a properties file (e.g., application.properties), so when it is changed/deleted (for some security reasons for example), we just change the properties file without touching the code

Reply:
This only for test purpose I put bobot as I could have put userExample